ihmc-state-estimation/src/main/java/us/ihmc/stateEstimation/humanoid/kinematicsBasedStateEstimation/IMUBasedPelvisRotationalStateUpdater.java

```diff
@@ -41,7 +41,7 @@ public class IMUBasedPelvisRotationalStateUpdater implements PelvisRotationalSta
    private final YoFrameVector3D yoRootJointAngularVelocity;
    private final YoFrameVector3D yoRootJointAngularVelocityInWorld;
 
-   private final BooleanParameter zeroYawAtInitialization = new BooleanParameter("zeroEstimatedRootYawAtInitialization", registry, true);
+   private final BooleanParameter zeroYawAtInitialization = new BooleanParameter("zeroEstimatedRootYawAtInitialization", registry, false);
```

Comment:
This and the other state estimator variable got changed by default. This in turn messed up every test of atlas at a non-zero initialization. You should probably go in and in yoru parameter file for the state estimator that uses this, change it back to true, but not edit the default value for your specific change.